test/ParseableInterface/conformances.swift

```diff
@@ -11,10 +11,69 @@ public protocol SimpleProto {
   func inference(_: Inferred)
 } // CHECK: {{^}$}}
 
-// CHECK-LABEL: public struct SimplImpl<Element> : SimpleProto {
-public struct SimplImpl<Element>: SimpleProto {
+// CHECK-LABEL: public struct SimpleImpl<Element> : SimpleProto {
+public struct SimpleImpl<Element>: SimpleProto {
   // NEGATIVE-NOT: typealias Element =
   // CHECK: public func inference(_: Int){{$}}
   public func inference(_: Int) {}
   // CHECK: public typealias Inferred = Swift.Int
 } // CHECK: {{^}$}}
+
+
+public protocol PublicProto {}
```

Comment:
Seems the `CHECK`s are missing here

Reply:
…right. I was manually inspecting the output but didn't actually write down the CHECK lines I wanted.